lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -518,6 +520,15 @@ export class BrowserCacheManager extends CacheManager {
         this.removeAccountKeyFromMap(key);
     }
 
+    /**
+     * Remove account entity from the platform cache if it's outdated
+     * @param accountKey
+     */
+    removeOutdatedAccount(accountKey: string): void {
```

Comment:
Can you use the existing removeAccount API instead? Should do the same things

Reply:
Unfortunately that method is async and also removes all tokens related to the account context, I'll try to find a way to factor out the part that removes the account object only.

Reply:
Ah thats right ok. Nvm this can stay.

Reply:
Can we rename it to removeAccountOnly() as a sync function and call it inside `removeAccount`? Can be done later, I think that is viable and removes code duplicity

Reply:
Name is fine. Just reusing is what I want to focus on.